README.md

```diff
@@ -1,44 +1,25 @@
 # VA.gov Team Repository
-*Last updated: [9/17/2019](#document-history)*
 
 This project aims to improve the lives of Veterans, their families, and their caregivers by facilitating the creation of interactive experiences on VA.gov on par with the private sector. It's comprised of a platform (the Veteran-facing Services Platform, aka VSP) which serves as the common framework for building cohesive products on VA.gov, and the VA.gov products themselves (the Veteran Facing Services, aka VFSs).
 
-This repository is used to store *non-code* documents and files created and used by teams building the VSP and VFSs.
+This repository is used to store *non-code* documents and files created and used by teams building the VSP and VFSs. It is also used for issue tracking for many parts of the VSP platform.
 
 ***An important note about security for contributors to this repository: this is a public, open source space! Which means it's great for collaborating, but decidedly not great for storing sensitive information (like credentials, PII, etc). So be mindful as you work in this repo. Don't store anything sensitive here!***
 
 For information on how to use this repo, see the [Repo Guidelines](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/orientation/repo-guidelines.md).
 
-#### Top level stats:
-
-- Number of VA.gov features live: 30something?
-- Number of VA.gov features in progress: 13?
-- Number of VFS Teams actively building VFSs: 13
-- Number of unique VA.gov user accounts: 2,984,036
-- VSP-hosted VA.gov page loads occuring in < 5s: 88.6%
-- Average uptime last 30 days: 99.1%
-
----------------
-
 ## How to navigate this repository
 
+### Knowledge hub
+The best starting place is the knowledge hub, that provides links to the content in this repository and elsewhere:
+https://department-of-veterans-affairs.github.io/va.gov-team/
+
 ### [/platform](./platform)
-All resources, templates, guides, and processes that support development of VFSs on VA.gov.
+All VSP resources, templates, guides, and processes that support development of VFSs on VA.gov.
```

Comment:
Not all platform resources are owned by VSP — for example, "research" is something that sits pretty squarely with DEPO, and "CMS" is included as part of the platform folder but is not part of VSP.

Reply:
gotcha, updatd. this. thanks!